project/AppDependencies.scala

```diff
@@ -3,35 +3,34 @@ import sbt._
 object AppDependencies {
   import play.sbt.PlayImport._
   
-  private val playVersion = "play-26"
+  private val playVersion = "play-28"
   
   private val pegdownVersion = "1.6.0"
 
   val compile = Seq(
     filters,
     ws,
-    "uk.gov.hmrc"       %% s"bootstrap-$playVersion" % "2.3.0",
-    "uk.gov.hmrc"       %% "domain"                  % s"5.10.0-$playVersion",
-    "uk.gov.hmrc"       %% "json-encryption"         % s"4.8.0-$playVersion",
-    "uk.gov.hmrc"       %% "mongo-caching"           % s"6.15.0-$playVersion" exclude("uk.gov.hmrc","time_2.11"),
-    "uk.gov.hmrc"       %% "auth-client"             % s"3.2.0-$playVersion",
-    "com.typesafe.play" %% "play-json-joda"          % "2.7.4"
+    "uk.gov.hmrc"       %% s"bootstrap-backend-$playVersion" % "5.3.0",
```

Comment:
bootstrap-backend-play can be upgraded to 5.7.0 now

Reply:
done